be/src/storage/lake/tablet_manager.cpp

```diff
@@ -254,16 +274,13 @@ StatusOr<TabletMetadataPtr> TabletManager::get_tablet_metadata(const string& pat
 Status TabletManager::delete_tablet_metadata(int64_t tablet_id, int64_t version) {
     auto location = tablet_metadata_location(tablet_id, version);
     _metacache->erase(location);
-    return fs::delete_file(location);
+    return ignore_not_found(fs::delete_file(location));
 }
 
-StatusOr<TabletMetadataIter> TabletManager::list_tablet_metadata(int64_t tablet_id, bool filter_tablet) {
+StatusOr<TabletMetadataIter> TabletManager::list_tablet_metadata(int64_t tablet_id) {
     std::vector<std::string> objects{};
     // TODO: construct prefix in LocationProvider
-    std::string prefix;
-    if (filter_tablet) {
-        prefix = fmt::format("{:016X}_", tablet_id);
-    }
+    std::string prefix = fmt::format("{:016X}_", tablet_id);
```

Comment:
is there any usecase to not filter tablets? looks like you change the behavior to always filtering.

Reply:
no usecase, I removed this useless feature